src/ml_analysis.py

```diff
@@ -291,6 +361,9 @@ def generate_csv_and_figure_reports(arr, csv_filepath, figure_filepath):
                      titleFontSize=15).\
     save(figure_filepath)
 
+    # Saving confusion matrix
+    save_confusion_matrix(names, cms, './reports/confusion_matrix.png')
```

Comment:
Should this be `../` at the front?

Reply:
@fsywang  This will work as it is a single(.), which points to the same repository which is the root repository

Reply:
@fsywang I assume that all the codes are run from the root directory. `./reports/` is the same as `reports/`